src/Controller/NominatimController.php

```diff
@@ -138,14 +178,14 @@ protected function search($input, int $count = 5, string $lang = 'en'){
       }
       else {
         // Means our call was correct, but nominatim failed. Could be temporary?
-          $results[] = [
-            'value' => NULL,
-            'label' => 'Sorry https://nominatim.openstreetmap.org responsed with an error. Please Try again'
-          ];
-        }
+        $results[] = [
+          'value' => NULL,
+          'label' => 'Sorry https://nominatim.openstreetmap.org responsed with an error. Please Try again'
+        ];
+      }
       return  $results;
     }
-    $this->messenger->addError(
+    \Drupal::messenger()->addError(
```

Comment:
Ok. Here is the messenger issue. So, did you test if $this->messenger was working?
To my knowledge, Abstract class Controller https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Controller%21ControllerBase.php/class/ControllerBase/8.8.x (and for 8.7.x too) is using the Messenger trait, which in turn: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Messenger%21MessengerTrait.php/function/MessengerTrait%3A%3Amessenger/8.8.x
defines $this->messenger. Was that not working? Please test and let me know. I feel we should just use $this->messenger is i'm correct there.

Reply:
@DiegoPino yes, I had an error saying something like, cannot call addError on null. Cleared my logs. Let me reproduce for you.

Reply:
@DiegoPino 

```
NOTICE: PHP message: Error: Call to a member function addError() on null in /var/www/html/web/modules/contrib/webform_strawberryfield/src/Controller/NominatimController.php on line 239 #0 /var/www/html/web/modules/contrib/webform_strawberryfield/src/Controller/NominatimController.php(168): Drupal\webform_strawberryfield\Controller\NominatimController->getRemoteJsonData('https://nominat...', Array)
#1 /var/www/html/web/modules/contrib/webform_strawberryfield/src/Controller/NominatimController.php(153): Drupal\webform_strawberryfield\Controller\NominatimController->processRequest('https://nominat...', Array)
#2 /var/www/html/web/modules/contrib/webform_strawberryfield/src/Controller/NominatimController.php(90): Drupal\webform_strawberryfield\Controller\NominatimController->reverse('latitude', '11.33989', 'en')
#3 /var/www/html/web/modules/contrib/webform_strawberryfield/src/Element/WebformNominatim.php(304): Drupal\webform_strawberryfield\Controller\NominatimController->handleRequest(Object(Symfony\Component\HttpFoundation\Request), 'reverse', 5, 'en')
```

Reply:
@marlo-longley so weird. Wonder if that is related to the fact that you are calling the request directly without initializing the Controller? And since the `trait` is part of the class, its not being initialized? Or maybe I need to call the trait differently. let me denug this, give me an hour or so. Thanks for reporting this!

Reply:
Ok, now i get it: it needs to be $this->messenger()->addError(). Can you give it a shot? (took me 2 minutes!) jajaja

Reply:
@DiegoPino thank you. Will fix and push.